acdc.js

```diff
@@ -1,7 +1,18 @@
 ( async function ( mw, $ ) {
 	'use strict';
 
-	const require = await mw.loader.using( [
+	let ClaimGuidGenerator,
+		Statement,
+		Claim,
+		PropertyNoValueSnak,
+		StatementListDeserializer,
+		StatementSerializer,
+		StatementDeserializer,
+		StatementWidget,
+		AddPropertyWidget;
+
+	try {
```

Comment:
You don't need to do try catch here, wikibase.serialization and wikibase.datamodel has been introdced recently and are deployed to production for a week already. You can simply switch to catch part without any issues (TM).

Reply:
Thanks, done.